ZelloChannelKit/ZelloChannelKit/ZCCSession.m

```diff
@@ -105,7 +105,7 @@ @interface ZCCSession () <ZCCImageMessageManagerDelegate, ZCCSocketDelegate, ZCC
 
 @property (nonatomic, strong, readonly, nonnull) ZCCQueueRunner *runner;
 
-@property (nonatomic, copy, readonly) NSString *authToken;
+@property (nonatomic, copy, readonly, nullable) NSString *authToken;
```

Comment:
does the default value for `_authToken` need to be set to an empty string on line 126 below, similar to username and password?

Reply:
I don't believe so, and signing in to the channel works correctly when I omit the auth token from the login call (the token key is omitted if the token is null)